### PR TITLE
Updated Legend

### PR DESCRIPTION
**Pull request by Rajandeep98**
Updated Information

### CODE REVIEW

File: apps/application-status/package.json
Comment (bolyachevets) [inline apps/application-status/package.json:1]: need to update version: https://github.com/bcgov/sbc-producthub/blob/4480215ecd5ac0eb9a7c74634d4bfe0dcf2b6583/apps/application-status/package.json#L3

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://application-status-dev--pr-56-80n7ota1.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://release-notes-dev--pr-56-o6wnsark.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://application-status-dev--pr-56-80n7ota1.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://release-notes-dev--pr-56-o6wnsark.web.app

**Pull request merged**